redfish-core/lib/log_services.hpp

```diff
@@ -2167,10 +2159,26 @@ inline void
                 messages::resourceNotFound(asyncResp->res, "LogEntry", entryId);
                 return;
             }
-            // TODO Handle for specific error code
+
+            const sd_bus_error* dbusError = msg.get_error();
+            if (dbusError == nullptr)
+            {
+                messages::internalError(asyncResp->res);
+                return;
+            }
+            if (std::string_view(
+                    "xyz.openbmc_project.Common.Error.Unavailable") ==
+                dbusError->name)
+            {
+                messages::propertyValueExternalConflict(asyncResp->res,
+                                                        "LogEntry", "Delete");
+                return;
+            }
+
             BMCWEB_LOG_ERROR("EventLogEntry (DBus) doDelete "
```

Comment:
Do we want to move the BMCWEB_LOG_ERROR() up so we will see it regardless of if the dbusError() provided anything else? (Or at add another BMCWEB_LOG_ERROR() if dbusError == nullptr?

Reply:
So, `if (dbusError == nullptr)` hits, we have the internalError, which is already in the journal. I feel that adding an additional trace there doesn't buy us anything. 

`if (ec.value() == EBADR)` or ` if (std::string_view("xyz.openbmc_project.Common.Error.Unavailable") == dbusError->name)` hit, they are 400 errors / client errors and based on https://github.com/openbmc/bmcweb/blob/06b71375b21fedf20c189965cb4e6925dfcbb044/DEVELOPING.md?plain=1#L227 
if there is an trace there, it should be a warning, I don't see a warning trace there helps us because we already have a unique return code

Reply:
Let me know if you want to discuss more